examples/terraform-aws-s3-example/policies.tf

```diff
@@ -0,0 +1,29 @@
+data "aws_iam_policy_document" "s3_bucket_policy" {
```

Comment:
Just like the locals, can you add this to `main.tf`?

Reply:
Implemented